loop.js

```diff
@@ -0,0 +1,13 @@
+const test = n => n > 0;
+const update = n => n - 1;
+
+// function loop that works as for loop statment, but use recursion instead
+const loop = (currentValue, test, update, body) => {
```

Comment:
And now, you have access to `test` and `update` functions in your scope so you can omit to pass those as arguments :)